libkeepass/kdb4.py

```diff
@@ -90,7 +90,7 @@ def write_to(self, stream):
         
         :arg stream: A writeable file-like object or IO buffer.
         """
-        if not (isinstance(stream, io.IOBase) or isinstance(stream, file)):
+        if not (isinstance(stream, io.IOBase)) or (not IS_PYTHON_3 and not isinstance(stream, file)):
```

Comment:
@pschmitt Indeed, that's even better as this avoids code duplication. Btw, do you think we still need to support Python2? Scripts compatible with both versions become hard to maintain (from my young experience). :-/